packages/vite/bin/vite.js

```diff
@@ -28,7 +28,7 @@ if (debugIndex > 0) {
       .map((v) => `vite:${v}`)
       .join(',')
   }
-  process.env.DEBUG = value
+  process.env.DEBUG = `${process.env.DEBUG || ''},${value}`
```

Comment:
We should only add the comma when needed
```suggestion
  process.env.DEBUG = `${process.env.DEBUG ? process.env.DEBUG + ',' : ''}${value}`
```